roles/mongodb/defaults/main.yml

```diff
@@ -2,3 +2,25 @@
 # MongoDB version to install
 # Should be '3.2' or '3.4'
 mongodb_version: "3.4"
+
+# do not enable mongo auth by default. Make sure to add required users in mongodb_users if auth is enabled.
+mongodb_enable_auth: no
+
+# the directory used to store generated credentials in
+mongodb_creds_dir: '.'
+
+# when adding auth, the login credentials to use
+mongodb_host: 127.0.0.1
+mongodb_port: 27017
+mongodb_admin_username: admin
+mongodb_admin_password: "{{ lookup('password', '{{ mongodb_creds_dir }}/mongodb-' + inventory_hostname + '-' + mongodb_username + ' length=42' ) }}"
```

Comment:
It seems like not ideal option to save mongo admin password in plain text.
Need to think about the alternative approaches/directions.

Reply:
dropped the password generation for now.